api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirEncounterServiceImpl.java

```diff
@@ -73,6 +74,74 @@ public Encounter get(@Nonnull String uuid) {
 		return result;
 	}
 	
+	@Override
+	public Encounter create(@Nonnull Encounter encounter) {
+		
```

Comment:
```suggestion
```

Reply:
Can you please generally remove the blank lines at the start and end of functions?

Reply:
@ibacher I will remove them.